### ISSUE TITLE
Latest version of deform doesn't work well with kotti_tinymce

### ISSUE THREAD

**Issue by nightmarebadger** [Kotti/kotti_tinymce#14]
If you update to the latest of deform, it breaks the default Document in Kotti - the TinyMCE doesn't appear. This happens because Kotti, by default, uses the `advanced` theme, but should use the `modern` theme. Till now it worked because our `richtext.pt` file in has a [workaround that changed None, "simple" and "advanced" to "modern"](https://github.com/Kotti/kotti_tinymce/blob/master/kotti_tinymce/templates/deform/richtext.pt#L32).

The commit that breaks the compatibility is here: https://github.com/Pylons/deform/commit/d159ee87d5837e86939eb4f31aa0b6840e5eaf06. 

With this commit, deform went from manually specifying each option in the TinyMCE.init() to an arbitrary `options` attribute. 

We need to update the `richtext.pt` to reflect that, but with that, we cannot use a simple workaround anymore. I've managed to come up with a workaround, but it's ugly. [Edit: not so ugly anymore, managed to use json.loads() to get the data] 

So basically, we're left with two options:
- Keep the workaround, preferably coming up with a better one
- Remove the workaround, change the `Document` content type in `Kotti` to use `modern` instead of `advanced` - everyone that currently uses `simple`, `advanced` or didn't define a theme will have to manually set the theme to `modern`
